### PR TITLE
dev-python/cython: version bump and cosmetic fixes.

### PR DESCRIPTION
**Pull request by netpositive36**

### CODE REVIEW

File: dev-python/cython/cython_python-0.28.2.recipe [hunk 1]
```diff
@@ -3,11 +3,11 @@ DESCRIPTION="Cython is an optimising static compiler for both the Python \
...
 HOMEPAGE="http://cython.org/"
-COPYRIGHT="2007-2017 Stefan Behnel, Robert Bradshaw, et al."
+COPYRIGHT="2007-2018 Stefan Behnel, Robert Bradshaw, et al."
 LICENSE="Apache v2"
 REVISION="1"
 SOURCE_URI="https://github.com/cython/cython/archive/$portVersion.tar.gz"
-CHECKSUM_SHA256="648feb8a257574a3e4ef854475e7d767732ef26e870d2f9fcc1ca5b244b3ac89"
+CHECKSUM_SHA256="d8fad58f7484466ff64a3d54f76943a3e2aef72a4efa1506eccca6ddc3ee547a"
 
 SOURCE_DIR="cython-$portVersion"
```
Comment (alaviss) [inline dev-python/cython/cython_python-0.28.2.recipe]: Could you also fix some cosmetic issues within the recipe as well?

There should be no blank lines between `CHECKSUM_SHA256` and `SOURCE_DIR`, please refer to [Guidelines#Ordering][0] for more details.

Archives should be in `<port name>-<port version>` format as in [Guidelines#Archive name][1].

```diff
-
+SOURCE_FILENAME="cython-$portVersion.tar.gz"
 SOURCE_DIR="cython-$portVersion"
```

A similar fix should also be done for the python3 version.

[0]: https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#ordering
[1]: https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#archive-name

Comment (netpositive36) [reply dev-python/cython/cython_python-0.28.2.recipe]: Thanks for the comments.

File: dev-python/cython/cython_python-0.28.2.recipe [hunk 2]
```diff
@@ -3,12 +3,12 @@ DESCRIPTION="Cython is an optimising static compiler for both the Python \
...
 HOMEPAGE="http://cython.org/"
-COPYRIGHT="2007-2017 Stefan Behnel, Robert Bradshaw, et al."
+COPYRIGHT="2007-2018 Stefan Behnel, Robert Bradshaw, et al."
 LICENSE="Apache v2"
 REVISION="1"
 SOURCE_URI="https://github.com/cython/cython/archive/$portVersion.tar.gz"
-CHECKSUM_SHA256="648feb8a257574a3e4ef854475e7d767732ef26e870d2f9fcc1ca5b244b3ac89"
-
+CHECKSUM_SHA256="d8fad58f7484466ff64a3d54f76943a3e2aef72a4efa1506eccca6ddc3ee547a"
+SOURCE_FILENAME="cython_python-$portVersion.tar.gz"
```
Comment (fbrosson) [inline dev-python/cython/cython_python-0.28.2.recipe]:
```diff
-SOURCE_FILENAME="cython_python-$portVersion.tar.gz"
+SOURCE_FILENAME="cython-$portVersion.tar.gz"
```

File: dev-python/cython/cython_python3-0.28.2.recipe [hunk 3]
```diff
@@ -3,12 +3,12 @@ DESCRIPTION="Cython is an optimising static compiler for both the Python \
...
 HOMEPAGE="http://cython.org/"
-COPYRIGHT="2007-2017 Stefan Behnel, Robert Bradshaw, et al."
+COPYRIGHT="2007-2018 Stefan Behnel, Robert Bradshaw, et al."
 LICENSE="Apache v2"
 REVISION="1"
 SOURCE_URI="https://github.com/cython/cython/archive/$portVersion.tar.gz"
-CHECKSUM_SHA256="648feb8a257574a3e4ef854475e7d767732ef26e870d2f9fcc1ca5b244b3ac89"
-
+CHECKSUM_SHA256="d8fad58f7484466ff64a3d54f76943a3e2aef72a4efa1506eccca6ddc3ee547a"
+SOURCE_FILENAME="cython_python3-$portVersion.tar.gz"
```
Comment (fbrosson) [inline dev-python/cython/cython_python3-0.28.2.recipe]:
```diff
-SOURCE_FILENAME="cython_python3-$portVersion.tar.gz"
+SOURCE_FILENAME="cython-$portVersion.tar.gz"
```

**Pull request merged**